torch_xla/distributed/xla_dist.py

```diff
@@ -150,10 +183,15 @@ def _run_remote_cmd(self, cmd, client_worker, shell=True, log=True):
     if log:
       self._stream_logs(proc, client_worker)
     proc.wait()
+    if proc.returncode == 255:
```

Comment:
Where does this 255 come from?
I would stick with an error code lower than 128 since negative (bit7 set) numbers are signals termination numbers.

Reply:
SG!